### PR TITLE
Plat UI 1170 add tests

### PR DESCRIPTION
**Pull request by matthewmascord**

### CODE REVIEW

File: src/main/resources/javascripts/src/mount.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+function mount(content) {
+  const dom = new JSDOM(content);
+  const { window } = dom;
+  const { document } = window;
+
+  return document.documentElement;
+}
+
+module.exports = mount;
```
Comment (oscarduignan) [inline src/main/resources/javascripts/src/mount.js:13]: possible small (untested) refactoring

```suggestion
const { JSDOM } = require('jsdom');

module.exports = content => new JSDOM(content).window.document.documentElement;
```

**Comment by matthewmascord** [comment]
retest this please

File: src/main/resources/javascripts/src/mount.js (hunk 1 above)
Comment (matthewmascord) [reply src/main/resources/javascripts/src/mount.js:13]: Thanks @oscarduignan. If I'm honest, I prefer the version with the named function and the de-structuring - it's just a little readable for me! 😅

**Comment by matthewmascord** [comment]
retest this please

**Review by oscarduignan — APPROVED** [review]

File: src/main/resources/javascripts/src/mount.js (hunk 1 above)
Comment (oscarduignan) [reply src/main/resources/javascripts/src/mount.js:13]: (sorry didn't see the notification for this yesterday)

**Pull request merged**